lib/chef/application/client.rb

```diff
@@ -384,6 +338,125 @@ def run_application
   end
 
   private
+
+  def configure_local_mode
+    update_local_mode
+    update_chef_repo_path
+    fetch_local_mode_recipes!
+  end
+
+  def verify_audit_mode!
+    mode = config[:audit_mode] || Chef::Config[:audit_mode]
+
+    return unless mode
```

Comment:
return from the middle of a method is a code smell, this should be `if mode ... end`